lib/Runtime/Library/JavascriptRegularExpression.cpp

```diff
@@ -124,6 +124,22 @@ namespace Js
         JavascriptError::ThrowTypeError(scriptContext, JSERR_This_NeedRegExp, propertyName);
     }
 
+    JavascriptString* JavascriptRegExp::GetFirstStringArg(Arguments& args, ScriptContext* scriptContext)
+    {
+        if (args.Info.Count == 1)
+        {
+            return scriptContext->GetLibrary()->GetUndefinedDisplayString();
+        }
+        else if (JavascriptString::Is(args[1]))
+        {
+            return JavascriptString::FromVar(args[1]);
```

Comment:
This was used in a few places in this file and I extracted it to a function. I'll keep it as it is for now, but will look into why we have the `JavascriptString::Is` check later.